eZ/Bundle/EzPublishCoreBundle/Resources/config/routing/internal.yml

```diff
@@ -1,14 +1,7 @@
 # Internal routes
-_ezpublishLocation:
-    path: /content/location/{locationId}/{viewType}/{layout}
-    defaults:
-        _controller: ez_content:viewLocation
-        viewType: full
-        layout: true
```

Comment:
That is a big BC break, even though not so many people may use it. I'd keep it for backport.